### ISSUE TITLE
Symbols loaded in `prelude_bazel` aren't loaded into all BUILD.bazel files

### ISSUE THREAD

**Issue by arie-stripe** [withered-magic/starpls#343]
Thank you for solving https://github.com/withered-magic/starpls/issues/337 so quickly!

## Issue

There's one other issue we're having with the `prelude_bazel` file now that v0.1.19 is released.

We use the `prelude_bazel` file to load our own macro wrappers for common Bazel rule kinds, and unfortunately starpls doesn't yet treat the symbols that are loaded into `prelude_bazel` as if they're loaded in all BUILD.bazel files (which is how it works with Bazel).

For example, say we have a macro in `//foo/bar:baz.bzl` named `java_library` that takes additional attributes and then creates a native `java_library`. Our `prelude_bazel` file would then look like this:

```starlark
load("//foo/bar:baz.bzl", "java_library")
```

However, starpls doesn't recognize this.

It also isn't propagating loads for non-overloaded targets - that's a red herring. Like no matter what macro is loaded in the prelude, that load isn't happening in BUILD.bazel files.

## Workaround

We currently have a workaround for this so we're not blocked. In order to get it to work, we add self-assignments for loaded symbols at the bottom of the prelude like this:

```starlark
load("//foo/bar:baz.bzl", "java_library")

java_library = java_library
```

Thanks for your help!

**Comment by withered-magic** [comment]
Just landed a fix as part of https://github.com/withered-magic/starpls/pull/345, will release v0.1.20 later today!

**Comment by withered-magic** [comment]
Just released v0.1.20 which includes this fix!
